src/app/features/dashboard/dashboard.component.ts

```diff
@@ -46,16 +48,16 @@ export class DashboardComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.authService.isOnAdminScreen = false;
     this.workplace = this.establishmentService.primaryWorkplace;
-    const workplaceUid: string = this.workplace ? this.workplace.uid : null;
-    this.canViewBenchmarks = this.permissionsService.can(workplaceUid, 'canViewBenchmarks');
-    this.canViewListOfUsers = this.permissionsService.can(workplaceUid, 'canViewListOfUsers');
-    this.canViewListOfWorkers = this.permissionsService.can(workplaceUid, 'canViewListOfWorkers');
-    this.canViewEstablishment = this.permissionsService.can(workplaceUid, 'canViewEstablishment');
-    this.canDeleteEstablishment = this.permissionsService.can(workplaceUid, 'canDeleteAllEstablishments');
+    this.workplaceUid = this.workplace ? this.workplace.uid : null;
```

Comment:
You can still have a type definition here if you want 
```suggestion
    this.workplaceUid: string | null = this.workplace ? this.workplace.uid : null;
```

Reply:
It's not being declared here though, it's got the type on line 33, is that ok?

Reply:
@duncanc19 think it just needs the `| null` on the type to communicate it can also be set to null